workflow/rules/common.smk

```diff
@@ -60,11 +60,25 @@ if config.get("trimmer_software", "None") == "fastp_pe":
         "prealignment/fastp_pe/{sample}_{flowcell}_{lane}_{barcode}_{type}_fastq1.fastq.gz",
         "prealignment/fastp_pe/{sample}_{flowcell}_{lane}_{barcode}_{type}_fastq2.fastq.gz",
     ]
+    alignment_input_fq1 = lambda wilcards: [
```

Comment:
Why do we need this? Is it not possible to reuse the `alignment_input`? It is pointing to the same files if I am not mistaken.